SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -66,43 +66,55 @@ void USpatialSender::Init(USpatialNetDriver* InNetDriver)
 	ClassInfoManager = InNetDriver->ClassInfoManager;
 }
 
+const WorkerAttributeSet ServerWorkerAttributeSet = { SpatialConstants::ServerWorkerType };
+const WorkerAttributeSet ClientWorkerAttributeSet = { SpatialConstants::ClientWorkerType };
+
+const WorkerRequirementSet ServerWorkerOnlyRequirementSet = { ServerWorkerAttributeSet };
+
 Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 {
 	AActor* Actor = Channel->Actor;
 	UClass* Class = Actor->GetClass();
 
 	FString ClientWorkerAttribute = improbable::GetOwnerWorkerAttribute(Actor);
 
-	WorkerAttributeSet ServerAttribute = { SpatialConstants::ServerWorkerType };
-	WorkerAttributeSet AIWorkerAttribute = { FString(TEXT("AIWorker")) };
-	WorkerAttributeSet ClientAttribute = { SpatialConstants::ClientWorkerType };
-	WorkerAttributeSet OwningClientAttribute = { ClientWorkerAttribute };
+	TArray<FString> ServerWorkerTypes = GetDefault<USpatialGDKSettings>()->ServerWorkerTypes;
 
-	WorkerRequirementSet ServersOnly = { ServerAttribute };
-	WorkerRequirementSet AllServerTypes = { ServerAttribute, AIWorkerAttribute };
-	WorkerRequirementSet ClientsOnly = { ClientAttribute };
-	WorkerRequirementSet OwningClientOnly = { OwningClientAttribute };
+	WorkerRequirementSet AllServerTypesRequirementSet;
```

Comment:
I suggest renaming this to `AnyServerRequirementSet` for consistency with the other two requirement sets.